README.md

```diff
@@ -1 +1,67 @@
-js-capstone
+# Group Capstone
+
+This is the first time at Microverse that we will build a project in a group. Apart from technical knowledge, we will need to use a few new processes and tools. We have been preparing for that with a few individual tasks in the Microverse program, and it is a time to connect all of them together. Besides - it's time to check what exactly we will build!
+
+## Built With
+
+- Visual Studio Code and GitHub.
+- HTML, CSS and JavaScript.
+- Linters for HTML, CSS and JavaScript.
+- WebPack and Gitflow.
+
+## Getting Started
```

Comment:
- [ ] [OPTIONAL] Kindly note that it is more professional to state some set of instructions on how to run this app locally. For example: Provide the URL of the project to get it cloned, direct a new user to checkout to the `develop` branch, then run:
```
npm i
```
```
npm run build
```
```
npm start
```